stock-tracker/src/store/subscriptions.js

```diff
@@ -0,0 +1,60 @@
+import {
+  setChartDataAction,
+  setChartErrorAction
+} from "../features/chart/redux/actions";
```

Comment:
Note: These imports show the nicety of having an `index.js` file in the feature root, which defines an "API" of the folder and exports the component, reducer, middleware, and externally required actions.
The import would then just be `"../features/chart";`